votor/src/certificate_pool_service.rs

```diff
@@ -201,10 +224,12 @@ impl CertificatePoolService {
                 &cert_pool,
                 &mut ctx,
                 &mut events,
+                &mut stats,
             );
 
             if standstill_timer.elapsed() > STANDSTILL_TIMEOUT {
                 events.push(VotorEvent::Standstill(highest_finalized_slot));
+                CertificatePoolServiceStats::incr_u16(&mut stats.standstill_reset);
```

Comment:
We don't need a separate metric for `standstill_reset` since it resets everytime we finalize a block which the `new_finalized_slot` metric reports.
Can call this just `standstill` and only update it here.

Reply:
Hmm, I'm confused. I think on this line you pointed out we also reset standstill timer without a new finalized slot, this is purely based on timer expiration?

Reply:
you reset the standstill timer if either:
1) We see a new finalized slot
2) We reach 10 seconds and notify the event loop

(1) is not interesting and is covered by the `new_finalized_slot` metric. (2) is very interesting and should be put in a metric either here or in the event loop.

(1) + (2) together in one metric doesn't tell us anything. Does that make sense?

Reply:
Sure, in that case I think I shouldn't increment standstill in case 1) at all, changed.